MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/usecase/signin/SigninActivity.java

```diff
@@ -4,10 +4,14 @@
 import androidx.lifecycle.Observer;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Intent;
```

Comment:
nit: Normally we only use blank space to separate import and import static.

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing